LR_Tourist/DA/Services/Repository/IRepository.cs

```diff
@@ -0,0 +1,24 @@
+﻿using System;
+using System.Collections.Generic;
+
+namespace DA.Services.Repository
+{
+    public interface IRepository<T>
+        where T : class
```

Comment:
T should be a IBaseEntity

Reply:
why?

Reply:
Because your Get and Delete method requires `Id` field.
And `T` type should have it.
IEntityBase have this requirement.